package.json

```diff
@@ -17,6 +17,7 @@
   "dependencies": {
     "async": "=2.4.1",
     "bignumber.js": "=4.0.2",
+    "bluebird": "=3.5.0",
```

Comment:
Yes, but can be shared with websockets (which uses different promise library, but can be switched to bluebird).